integration/specs/Accordion/accordion-9.spec.js

```diff
@@ -0,0 +1,25 @@
+const PageAccordion = require('../../../src/components/Accordion/pageObject');
+const { ARROW_DOWN_KEY, ENTER_KEY } = require('../../constants');
+
+const ACCORDION = '#accordion-9';
+
+const addAdvancedSettings = () => $('#button-icon_add-new-advanced-settings').click();
+
+describe('Accordion with AccordionOption changed dynamically', () => {
+    it('should select the new option with keyboard after it is added dynamically', () => {
+        browser.url('/#!/Accordion/9');
+        const component = $(ACCORDION);
+        component.waitForExist();
+        const accordion = new PageAccordion(ACCORDION);
+        accordion.clickFirstButton();
```

Comment:
The accordion page object has a getItem method that returns a page object for each section child. Since we don't know how much sections will have an Accordion and we would need to click each one, instead of creating a clickFirstButton method you should do:
```
const firstSection = accordion.getItem(0);
firstSection.clickButtonIcon();
```
with this approach, you will be able to click whatever section you want